chapters/appendix-IV-SPDX-license-expressions.md

```diff
@@ -41,6 +41,14 @@ A valid `<license-expression>` string consists of either:
 
 There MUST NOT be whitespace between a license-id and any following `+`. This supports easy parsing and backwards compatibility. There MUST be whitespace on either side of the operator "WITH". There MUST be whitespace and/or parentheses on either side of the operators `AND` and `OR`.
 
+## Case sensitivity <a name="case-sensitivity"></a>
+
+License expression operators (`AND`, `OR` and `WITH`) should be matched in a *case-sensitive* manner.
+
+License identifiers used in SPDX documents or source code files should be matched in a *case-insensitive* manner. In other words, `MIT`, `Mit` and `mIt` should all be treated as the same identifier and referring to the same license.
```

Comment:
I checked and the SPDX tools look up the license by using the case sensitive URI - so implementing a case insensitive license ID would be somewhat substantial change to the SPDX tools (and any other library that uses the URL based on the ID - which could be quite a few tools since this has been the reccomended approach.

Recommend we change this to case sensitive ID's.

Reply:
Thanks @goneall -- I'm fine with changing it, but wonder if we should discuss on the tech team call today, since I'd understood the prior consensus at https://github.com/spdx/spdx-spec/issues/63#issuecomment-394805189 to be that they would be treated as case-insensitive.

Reply:
I just found a PR from Feb 2018 when this issue was first discussed: https://github.com/spdx/tools/pull/153/files  This PR makes the entire expression (including the and's and or's) case insensitive.

Based on this and the prior consensus, we can leave the license ID's case insensitive as proposed.  I'll merge in the PR to the tools.

It would be worth raising on the call since other tools may not be implementing the case sensitivity - just want to give a heads up to other tool writers they should check their parser implementations.